tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2647,4 +2647,60 @@ public function theJsonDataResponseShouldNotContainUserEmail(): void {
 			"Response contains email '$email' but should not."
 		);
 	}
+
+	/**
+	 * @Then user :byUser using password :password should be able to create a new user :user with default attributes
+	 *
+	 * @param string $byUser
+	 * @param string $password
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function userUsingPasswordShouldBeAbleToCreateANewUserWithDefaultAttributes(string $byUser, string $password, string $user): void {
+		$response = GraphHelper::createUser(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$byUser,
+			$password,
+			$user,
+			'',
+			null,
+			null
+		);
+		Assert::assertEquals(
+			201,
+			$response->getStatusCode(),
+			__METHOD__ . " cannot create user '$user' using Graph API.\nResponse:" .
+			json_encode($this->featureContext->getJsonDecodedResponse($response))
+		);
+		$this->featureContext->addUserToCreatedUsersList($user, '', null, null, null);
```

Comment:
others should be default
```suggestion
		$this->featureContext->addUserToCreatedUsersList($user);
```